src/clint.py

```diff
@@ -2600,16 +2600,23 @@ def CheckBraces(filename, clean_lines, linenum, error):
         else:
             func_start_linenum = end_linenum + 1
             while not clean_lines.lines[func_start_linenum] == '{':
-                if not Match(r'^(?:\s*\b(?:FUNC_ATTR|REAL_FATTR)_\w+\b(?:\(\d+(, \d+)*\))?)+$',
-                             clean_lines.lines[func_start_linenum]):
+                attrline = Match(r'^(\s*)(?:\b(?:FUNC_ATTR|FUNC_API|REAL_FATTR)_\w+\b(?:\(\d+(, \d+)*\))?\s*)+$',
+                                 clean_lines.lines[func_start_linenum])
+                if attrline:
+                    if len(attrline.group(1)) != 4:
+                        error(filename, func_start_linenum,
+                              'whitespace/indent', 5,
+                              'Function attribute line should be indented '
```

Comment:
Current message says “Function attribute line should be indented four spaces”. It looks like it is missing a preposition after “indented”, I would put “with” there.